packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -21,9 +23,11 @@ export default function ProfileButton({
         <button
           type="button"
           className="flex items-center p-0 ml-0.5 font-bold no-underline rounded-lg border-none cursor-pointer text-theme-label-primary bg-theme-bg-secondary typo-callout focus-outline"
-          onClick={onMenuClick}
+          onClick={onClick || onMenuClick}
```

Comment:
Why do we need both callbacks?

Reply:
We can actually removed this, Tsahi modified the design so this would no longer be needed.